src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -516,6 +585,27 @@ export class QuickBar extends LitElement {
           color: var(--primary-text-color);
         }
 
+        span.command-category {
+          font-weight: bold;
+          padding: 3px;
+          display: inline-flex;
+          border-radius: 6px;
+          color: black;
+        }
+
+        .command-category.reload {
+          --ha-chip-background-color: #cddc39;
```

Comment:
@bramkragten I committed your change here (changing `pink` to `#cddc39`) without checking what `#cddc39` was. Did you mean to choose a yellow here to replace the pink?

![image](https://user-images.githubusercontent.com/1480827/112355677-1f653000-8c8b-11eb-8e91-62eaa6dd6512.png)

Reply:
That's green right? 😄 Yeah, I thought it would match reload well?

Reply:
It's much closer to yellow ;)

green == ( 0, 255, 0 )
yellow == ( 255, 255, 0 )
#cddc39 == ( 205, 255, 57 )

Reply:
But yeah, I'm fine with it. For whatever reason, a pinkish color felt right to me so I guess it's just subjective. 🤷